test/unit/gateways/ixopay_test.rb

```diff
@@ -77,9 +82,26 @@ def test_failed_authorize
     assert_equal '2003', response.error_code
   end
 
-  def test_successful_capture; end
+  def test_successful_capture
+    @gateway.expects(:ssl_post).returns(successful_capture_response)
 
-  def test_failed_capture; end
+    response = @gateway.capture(@amount, 'auth-123')
```

Comment:
Consider changing this to something more similar to the actual authorization value passed in, e.g., `'00eb44f8f0382443cce5|20191028-00eb44f8f0382443cce5'`.

Reply:
Done.